pkgs/applications/virtualization/podman/disable-go-module-mode.patch

```diff
@@ -0,0 +1,31 @@
+From b4e132f75b2ada1c098c02e951aade892a281ea4 Mon Sep 17 00:00:00 2001
+From: Mario Rodas <marsam@users.noreply.github.com>
+Date: Fri, 13 Dec 2019 04:20:00 -0500
+Subject: [PATCH] Do not check Go module-mode availability
+
+We use buildGoPackage since podman vendorizes its dependencies. The
+module-mode check will be always be true because nixpkgs uses Go 1.13 by
+default, and building go modules with buildGoPackage may lead to
+impurities.
+---
+ Makefile | 4 ----
+ 1 file changed, 4 deletions(-)
+
+diff --git a/Makefile b/Makefile
+index a5265653..1585d04c 100644
+--- a/Makefile
++++ b/Makefile
+@@ -34,10 +34,6 @@ BUILDTAGS ?= \
+ 	varlink
+ 
+ GO_BUILD=$(GO) build
+-# Go module support: set `-mod=vendor` to use the vendored sources
```

Comment:
The `-mod=vendor` should ensure that we do always use the vendored dependencies. So I’m wondering if we really need this patch at all.

Reply:
interesting, I assumed that GO111MODULE=on made network requests for the checksum database, but it does not seem the case here. 
I've removed the patch, thanks!